src/ContainerRenameModal.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import {
+    Button,
+    Form, FormGroup,
+    Modal, TextInput
+} from '@patternfly/react-core';
+import cockpit from 'cockpit';
+
+import * as client from './client.js';
+
+import { ErrorNotification } from './Notification.jsx';
+import * as dockerNames from 'docker-names';
+
+const _ = cockpit.gettext;
+
+class ContainerRenameModal extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            name: dockerNames.getRandomName(),
```

Comment:
The dialog input should start with the *current* container name. Often you merely want to tweak it a bit (perhaps add a suffix). I don't think we should use random names here at all -- the whole point of renaming is to give it some descriptive user-selected name.

Reply:
I agree, starting editing with the old container name makes more sense to me.